agentops_dcp.go

```diff
@@ -50,6 +67,117 @@ type VbSeqNoEntry struct {
 // GetVBucketSeqnosCallback is invoked with the results of `GetVBucketSeqnos` operations.
 type GetVBucketSeqnosCallback func([]VbSeqNoEntry, error)
 
+// OpenStream opens a DCP stream for a particular VBucket.
+func (agent *Agent) OpenCollectionStream(vbId uint16, flags DcpStreamAddFlag, vbUuid VbUuid, startSeqNo,
```

Comment:
I think that to not have 2 functions then we'll have to release a new major? I think that we can roll the 2 into 1 and support collections and older clients but the function signature needs to change.